pkg/metrics/metrics.go

```diff
@@ -41,6 +41,20 @@ var BuildInfo = prometheus.NewGaugeFunc(
 	func() float64 { return 1 },
 )
 
+// DbgMetricHoldTime is Dbg metric hold time,
+// after that, Dbg metric will be cleaned up in case
+// more and more metrics exists.
+const DbgMetricHoldTime = 3 * 60
+const metricWithCustomLabels = "MetricsWithCustomLabels"
+
+// Killing is the delimiter keyword used to distinguish if pod event is for current pod,
+// not for same name pod already killed, which is useful to calculate scheduling metrics.
+const Killing = "Killing"
+
+// FailedScheduling is the keyword to find out failed scheduling event, which is useful
+// to calculate  scheduling metrics.
+const FailedScheduling = "FailedScheduling"
```

Comment:
maybe you can name the const with prefix  EventReason,
EventReasonFailedScheduling.

Reply:
Agree with you, that naming would be better to understand.